bazelisk.py

```diff
@@ -258,6 +258,8 @@ def normalized_machine_arch_name():
     machine = platform.machine().lower()
     if machine == "amd64":
         machine = "x86_64"
+    if machine == "aarch64":
```

Comment:
my pending tensorstore patch uses `elif` here

Reply:
I updated my commit accordingly. Thanks for the tip. :)